core/controllers/topics_and_skills_dashboard.py

```diff
@@ -110,10 +110,24 @@ def get(self) -> None:
             topic_summary_dict['classroom'] = topic_classroom_dict.get(
                 topic_summary_dict['id'], None)
 
-        updated_topic_summary_dicts = (
-            topic_services.update_chapters_counts_in_topic_summaries(
+        topic_chapter_count_dicts = (
```

Comment:
chapter_counts_by_topic_id, or topic_ids_to_topic_chapter_counts (which reads better to you?)

The existing name is incorrect because there's only one dict here.

Reply:
Done